NODE_ENV 命令不兼容 win 平台

```
D:\Project\WebIDE\frontend>npm run build

> coding-web-ide@1.0.0 build D:\Project\WebIDE\frontend
> NODE_ENV=production ./node_modules/webpack/bin/webpack.js

'NODE_ENV' 不是内部或外部命令，也不是可运行的程序或批处理文件。

npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
npm ERR! node v6.5.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! coding-web-ide@1.0.0 build: `NODE_ENV=production ./node_modules/webpack/bin/webpack.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the coding-web-ide@1.0.0 build script 'NODE_ENV=production ./node_modules/webpack/bin/webpack.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the coding-web-ide package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     NODE_ENV=production ./node_modules/webpack/bin/webpack.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs coding-web-ide
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls coding-web-ide
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\Project\WebIDE\frontend\npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --port=8060",
-    "build": "NODE_ENV=production ./node_modules/webpack/bin/webpack.js",
+    "build": "cross-env NODE_ENV=production ./node_modules/webpack/bin/webpack.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "keywords": [
@@ -33,7 +33,8 @@
     "url-loader": "^0.5.7",
     "webpack": "^1.13.1",
     "webpack-dev-server": "^1.14.1",
-    "webpack-merge": "^0.14.1"
+    "webpack-merge": "^0.14.1",
+    "cross-env": "^2.0.1"
   },
   "dependencies": {
     "bootstrap-styl": "^5.0.7",

